### ISSUE TITLE
podcast_has_no_default_playlists_message used incorrectly

### ISSUE THREAD

**Issue by vlzuykov** [Podcast-Republic/PRApp#278]
When I swipe on the episode and activate Add to playlists menu and then don't choose any playlist, the app shows podcast_has_no_default_playlists_message, but in this case the reason is different. I think the app should either:
1. Display something like "No playlist selected" or
2. Delete this episode or episodes from all playlists if it was in any playlist.

**Comment by gemiren** [comment]
Agreed. The message should be changed to 'no playlist selected' if the action is not 'add to the default playlists'. Thanks!
